.gitmodules

```diff
@@ -4,5 +4,5 @@
     ignore = dirty
 [submodule "libraries/fc"]
     path = libraries/fc
-    url = https://github.com/bitshares/bitshares-fc.git
+    url = https://github.com/jmjatlanta/bitshares-fc.git
```

Comment:
If you need modifications to bitshares-fc, please do so in the other repo and not redirect the submodules in general!

Reply:
I will back this out. It will have to be remembered that this PR must be merged with the related PR for bitshares-fc